docs/source/asynchronous.rst

```diff
@@ -1,10 +1,9 @@
 Asynchronous Operation
 ======================
 
-Dask.distributed can operate as a fully asynchronous framework and so
-interoperate with other highly concurrent applications.  Internally Dask is
-built on top of Tornado coroutines but also has a compatibility layer for
-asyncio (see below).
+Dask run fully asynchronously and so interoperate with other highly concurrent
+applications.  Internally Dask is built on top of Tornado coroutines but also
+has a compatibility layer for asyncio (see below).
```

Comment:
Should mention the transparent creation of threads, if not using async, somewhere?

Reply:
No objection to mentioning thread creation within the docs.  I don't think that here is the place though (this is about running within async-await functions).  This hasn't come up in a user issue that I'm aware of, so it's somewhat far down my priority list at the moment.